src/timer.rs

```diff
@@ -87,14 +87,14 @@ impl CountDownTimer<SYST> {
     }
 }
 
-impl CountDown for CountDownTimer<SYST> {
-    type Time = Hertz;
+impl embedded_hal::timer::CountDown for CountDownTimer<SYST> {
+    type Time = MicroSecond;
```

Comment:
I realize Hertz -> MicroSecond is quite a big change, it felt more natural in my mind. However I am fine with reverting that...